pvlib/test/test_ivtools.py

```diff
@@ -102,6 +109,29 @@ def test_fit_sdm_cec_sam(get_cec_params_cansol_cs5p_220p):
                 cells_in_series=1, temp_ref=25)
 
 
+@requires_scipy
+def test_fit_sdm_desoto(get_test_specs_params):
+    result = ivtools.fit_sdm_desoto(cells_in_series=60,
+                                    **get_test_specs_params)
+    result_expected = {'I_L_ref': 9.452324509050774,
+                       'I_o_ref': 3.2246097466679494e-10,
+                       'a_ref': 1.5912875522463978,
+                       'R_sh_ref': 125.79869968976132,
+                       'R_s': 0.2978148476600744,
+                       'alpha_sc': 0.005658,
+                       'EgRef': 1.121,
+                       'dEgdT': -0.0002677,
+                       'irrad_ref': 1000,
+                       'temp_ref': 25}
+    assert np.allclose(pd.Series(result), pd.Series(result_expected),
+                       rtol=1e-4)
+    with pytest.raises(RuntimeError):
```

Comment:
Should check that the exception raised is the exception we're after.  Could do it like this (learned from @wholmgren quite recently):
```
with pytest.raises(RuntimeError) as exc:
    ivtools.fit….
    assert expected_text in str(exc.value)
```